website/docs/best-practices/how-we-mesh/mesh-4-faqs.md

```diff
@@ -0,0 +1,314 @@
+
+Used in conjunction with dbt Mesh, dbt Explorer becomes a powerful tool for visualizing and understanding the relationships and dependencies between models across multiple dbt projects.
+
+</detailsToggle>
+
+<detailsToggle alt_header="How does the dbt Cloud CLI relate to and work with dbt Mesh?">
+
+The [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) allows users to develop and run dbt commands from their preferred development environments, like VS Code, Sublime Text, or terminal interfaces. This flexibility is particularly beneficial in a dbt Mesh setup, where managing multiple projects can be complex. Developers can work in their preferred tools while leveraging the centralized capabilities of dbt Cloud.
+
+</detailsToggle>
+
+## Availability
+
+<detailsToggle alt_header="Does dbt Mesh require me to be on a specific version of dbt?">
+
+Yes — your account must be on at least dbt v1.6 to take advantage of [cross-project dependencies](/docs/collaborate/govern/project-dependencies), one of the most crucial underlying capabilities required to implement a dbt Mesh.
+
+</detailsToggle>
+
+<detailsToggle alt_header="Is there a way to leverage dbt Mesh capabilities in dbt Core?">
+
+Not all of them. While dbt Core defines several of the foundational elements for dbt Mesh, dbt Cloud offers an enhanced experience that leverages these elements for scaled collaboration across multiple teams, facilitated by multi-project discovery in dbt Explorer that’s tailored to each user’s individual access.
+
+Several of the key components that underpin the dbt Mesh pattern — including model contracts, versions, and access modifiers — are defined and implemented in dbt Core. We believe these are components of the core language, which is why their implementations are open source. We want to define a standard pattern that analytics engineers everywhere can adopt, extend, and help us improve.
+
+To reference models defined in another project, users can also leverage a longstanding feature of dbt Core: [packages](/docs/build/packages). By importing an upstream project as a package, dbt will import all models defined in that project, which enables the resolution of cross-project references to those models — [optionally restricted](/docs/collaborate/govern/model-access#how-do-i-restrict-access-to-models-defined-in-a-package) to just the models with `public` access.
+
+The major distinction comes with dbt Cloud's metadata service, which is unique to the dbt Cloud platform and allows for the resolution of references to only the public models in a project. This service enables users to take dependencies on upstream projects, and reference just their `public` models, *without* needing to load the full complexity of those upstream projects into their local development environment.
+
+</detailsToggle>
+
+<detailsToggle alt_header="Does dbt Mesh require a specific dbt Cloud plan?">
```

Comment:
This question should probably be higher up. The section on "Compatibility with other features" isn't the first place I'd look for it.

Reply:
This is under a distinct section called "Availability" to be clear!